Source/Fuse.Reactive.JavaScript/FuseJS/Builtins.uno

```diff
@@ -44,6 +45,7 @@ namespace Fuse.Reactive.FuseJS
 			new FileModule(import BundleFile("Diagnostics.js")).EvaluateExports(context, "FuseJS/Diagnostics");
 			
 			Observable = (Scripting.Function)new FileModule(import BundleFile("Observable.js")).EvaluateExports(context, "FuseJS/Observable");
+			TreeObservable = (Scripting.Function)new FileModule(import BundleFile("TreeObservable.js")).EvaluateExports(context, "FuseJS/TreeObservable");
```

Comment:
There is two layers. The low-level primitive is called `TreeObservable`, but the JS-facing API is called Store (which inherits TreeObservable)